### PR TITLE
[DOCS] EditorScript

### PR DESCRIPTION
**Pull request by zerastos**

### CODE REVIEW

File: doc/classes/EditorScript.xml [hunk 1]
```diff
@@ -1,10 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorScript" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Simple script to perform changes in the currently edited scene.
+		Base script that can be used to add extension functions to the editor.
 	</brief_description>
 	<description>
-		This script can be run from the Scene -&gt; Run Script menu option.
+		Scripts extending this class and implementing it's [code]_run()[/code] method can be executed from the Script Editor's [code]File -> Run[/code] menu option (or by pressing [code]CTRL+Shift+X[/code]) while the editor is running. This is useful adding custom in-editor functionality to Godot. For more complex additions, consider using [EditorPlugin]\ s instead. Note that extending scripts need to have [code]tool mode[/code] enabled.
```
Comment (StraToN) [inline doc/classes/EditorScript.xml]: Grammar: 
- "its", not "it's"
- This is useful **for** adding custom in-editor functionality to Godot
- "using [EditorPlugin]\ **s** instead" : one "s" alone
- (not sure) "Note that extending scripts need**s** to have..."

File: doc/classes/EditorScript.xml [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+		Example script:
+		[codeblock]
+		tool
+		extends EditorScript
+		
+		func _run():
+			print("Hello from the Godot Editor!")
+		[/codeblock]
+		Note that the script is run in the Editor context, which means the output is visible in the console window started with the Editor (STDOUT) instead of the usual Godot *Output* dock.
```
Comment (StraToN) [inline doc/classes/EditorScript.xml]: Note that the script is r**an**

File: doc/classes/EditorScript.xml [hunk 3]
```diff
@@ -23,18 +33,21 @@
 			<argument index="0" name="node" type="Node">
 			</argument>
 			<description>
+				Not yet implemented.
```
Comment (StraToN) [inline doc/classes/EditorScript.xml]: I'd leave this uncommented instead. This way, we will be able to detect the missing documentation when the implementation will be available.

**Review by StraToN — CHANGES_REQUESTED** [review]

File: doc/classes/EditorScript.xml (hunk 2 above)
Comment (akien-mga) [reply doc/classes/EditorScript.xml]: Nope, "is run" is correct. run/ran/run :)

Comment (StraToN) [reply doc/classes/EditorScript.xml]: Oops. Indeed, re-reading my BS right now. This happens when you review PRs at work :D

File: doc/classes/EditorScript.xml (hunk 1 above)
Comment (Fallayn) [reply doc/classes/EditorScript.xml]: Fixed the first two, "[EditorPlugin]\ s" should become "[EditorPlugin]s" once parsed, and the last one should be correct as is.

File: doc/classes/EditorScript.xml (hunk 3 above)
Comment (Fallayn) [reply doc/classes/EditorScript.xml]: Fixed.

**Comment by Fallayn** [comment]
@StraToN Done.

**Review by StraToN — APPROVED** [review]

**Pull request merged**